tests/test_views.py

```diff
@@ -0,0 +1,32 @@
+from django.test import TestCase, Client
+
+from taxi.forms import DriverSearchForm, ManufacturerSearchForm, CarSearchForm
+
+
+class TestSearchFunctionDriver(TestCase):
+    def test_driver_search_form_is_valid(self):
+        form_data = {
+            "username": "test user"
+        }
+        form = DriverSearchForm(data=form_data)
+        self.assertEqual(form.is_valid(), True)
+
+
+class TestSearchFunctionManufacturer(TestCase):
+    def test_manufacturer_search_form_is_valid(self):
+        form_data = {"name": "test manufacturer",
+                     "country": "test country",
```

Comment:
The `ManufacturerSearchForm` does not have a 'country' field according to the form definition. Remove 'country' from `form_data` to match the actual form fields.